modules/nf-core/blast/updateblastdb/main.nf

```diff
@@ -0,0 +1,50 @@
+process BLAST_UPDATEBLASTDB {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.15.0--pl5321h6f7f691_1':
+        'biocontainers/blast:2.15.0--pl5321h6f7f691_1' }"
+
+    input:
+    tuple val(meta)
```

Comment:
If meta is the only input passed, a tuple is not necessary. I am not sure if meta is even the correct name for this, as from what I understand it refers to database IDs rather than samples. Consider `val db_name` or something similar.

Reply:
Do you know any other similar module example by chance?